chia/full_node/mempool.py

```diff
@@ -492,7 +497,9 @@ async def create_bundle_from_mempool_items(
         eligible_coin_spends = EligibleCoinSpends()
         coin_spends: List[CoinSpend] = []
         sigs: List[G2Element] = []
-        log.info(f"Starting to make block, max cost: {self.mempool_info.max_block_clvm_cost}")
+        # Deduct the block overhead from the maximum block clvm cost limit
+        EFFECTIVE_MAX_BLOCK_CLVM_COST = self.mempool_info.max_block_clvm_cost - BLOCK_OVERHEAD
```

Comment:
it seems like it would be a lot simpler to just compute `self.mempool_info.max_block_clvm_cost` in a different way (i.e. deduct the block overhead)